src/main/views/pending-subscriptions.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./macros/common-components.njk" import submitButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+    text: text,
+    href: '#',
+    attributes: {
+      onClick: "history.back();event.preventDefault();"
+    }
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{ header }}</h1>
+  <h3 class="govuk-heading-s">{{ subheader }}</h3>
+<form action="subscription-confirmed"  method="post">
+  <table class="govuk-table govuk-!-width-one">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">{{ table.header1 }}</th>
+      <th scope="col" class="govuk-table__header">{{ table.header2 }}</th>
+      <th scope="col" class="govuk-table__header govuk-!-text-align-right">{{ table.header3 }}</th>
+      <th scope="col" class="govuk-table__header govuk-!-text-align-right">{{ table.header4 }}</th>
+    </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+    {% for hearing in searchResults %}
+      <tr class="govuk-table__row">
+        <td scope="row" class="govuk-table__cell--no-line">{{ hearing.urn }}</td>
+        <td scope="row" class="govuk-table__cell--no-line">{{ hearing.caseNumber }}</td>
+        <td class="govuk-table__cell--no-line govuk-!-text-align-right">{{ hearing.caseName }}</td>
+        <td class="govuk-table__cell--no-line govuk-!-text-align-right"><a href="/remove-case?id={{ hearing.hearingId }}">{{ removeText }}</a></td>
+      </tr>
+    {% endfor %}
+    </tbody>
+  </table>
+  {{ submitButton(buttonText) }}
+  <a class="govuk-link" href="{{ addOther_link }}">{{ addOther_text }}</a>
```

Comment:
The anchor tag needs updating to be centred to the right of the button (See pending-subscriptions.png screenshot in ticket). Currently it is displayed aligned to the top.

Reply:
test